client/my-sites/checkout/src/components/item-variation-picker/jetpack-variant-dropdown-price.tsx

```diff
@@ -0,0 +1,90 @@
+import formatCurrency from '@automattic/format-currency';
+import { useMobileBreakpoint } from '@automattic/viewport-react';
+import { hasCheckoutVersion } from '@automattic/wpcom-checkout';
+import { useTranslate } from 'i18n-calypso';
+import { Discount, Label, Price, PriceTextContainer, Variant } from './styles';
+import type { WPCOMProductVariant } from './types';
+import type { FunctionComponent } from 'react';
+
+const isFreeMonthTrial = ( variant: WPCOMProductVariant ) =>
```

Comment:
We might want to rename this `isFirstMonthTrial` because this would return `true` regardless of whether it's a free trial or a $1 trial or something else